js/index.js

```diff
@@ -1 +1,71 @@
 // Your code goes here
+
+// Const
+const logo = document.querySelector('.logo-heading');
+const anchors = document.querySelectorAll('a');
+const busImg = document.querySelectorAll('img');
+const topb = document.querySelector('.main-navigation');
+const buttons = document.querySelectorAll('.btn');
+
+// Change background color of navigation when 'dblclick'
+topb.addEventListener('dblclick', function(e) {
+    topb.style.backgroundColor = "#FFEBCD";
+})
+
+// Change header text on 'mouseover'
+logo.addEventListener('mouseover', function(e) {
+    logo.textContent = 'All aboard the PARTY Bus!';
+});
+
+// Changes header back to Fun Bus when 'mouseout'
+logo.addEventListener('mouseout', function(e) {
+    logo.textContent = 'Fun Bus';
+});
+
+
+// Change colors when you 'click' nav links
+anchors.forEach(function(e) {
+  e.addEventListener('click', function(change) {
+    change.target.style.color = "#17A2B8";
+  });
+});
+
+// Increases img width by 20% when 'mousedown'
+busImg.forEach(function(e) {
+    e.addEventListener('mousedown', function(down) {
+    down.target.style.maxWidth = '120%';
+    });
+});
+
+// Returns img width to normal when 'mouseup'
+busImg.forEach(function(e) {
+    e.addEventListener('mouseup', function(up) {
+    up.target.style.maxWidth = '';
+    });
+});
+
+// Changes button text when you 'click'
+buttons.forEach(function(index, i) {
```

Comment:
remember forEach takes a call back function. the first parameter in the call back function is the element the gets passed through. In this case, each button. The second parameter is the index.